src/screens/claimables/ClaimingTransactionClaimable.tsx

```diff
@@ -134,9 +139,61 @@ export const ClaimingTransactionClaimable = ({ claimable }: { claimable: Transac
     true
   );
 
+  const { mutate: claimClaimable } = useMutation({
+    mutationFn: async () => {
+      if (!txPayload) {
+        setClaimStatus('error');
+        logger.error(new RainbowError('[ClaimingTransactionClaimable]: Failed to claim claimable due to missing tx payload'));
+        return;
+      }
+
+      const wallet = await loadWallet({
+        address: accountAddress,
+        showErrorIfNotLoaded: false,
+        provider,
+      });
+
+      if (!wallet) {
+        // Biometrics auth failure (retry possible)
+        setClaimStatus('error');
+        return;
+      }
+
+      const { errorMessage } = await walletExecuteRap(wallet, {
+        type: 'claimTransactionClaimableRap',
+        claimTransactionClaimableActionParameters: { claimTx: txPayload, asset: claimable.asset },
```

Comment:
@walmat you already reviewed this fn in previous PR, it was actually meant to be added in this PR so i moved it here. the only difference is this new param `asset` that's passed to the walletExecuteRap. it's used to generate the pending tx.